packages/formstr-app/src/containers/CreateFormNew/components/QuestionCard/InputElements/OptionTypes/CheckBoxCreator.tsx

```diff
@@ -27,30 +28,40 @@ export const CheckboxCreator: React.FC<CheckboxCreatorProps> = ({
       {choices?.map((choice) => {
         let [choiceId, label, settingsString] = choice;
         let settings = JSON.parse(settingsString || "{}") as ChoiceSettings;
+        const isOtherOption = settings.isOther;
+        
         return (
           <div className="radioButtonItem" key={choiceId}>
             <Checkbox disabled key={choiceId + "checkbox"} />
             <Input
               key={choiceId + "input"}
-              defaultValue={label}
+              defaultValue={isOtherOption ? "Other" : label}
               onChange={(e) => {
                 handleLabelChange(
                   e.target.value,
-                  choiceId!,
+                  choiceId,
                   choices,
                   handleNewChoices
                 );
               }}
-              placeholder="Enter an option"
+              placeholder={isOtherOption ? "Other" : "Enter an option"}
               className="choice-input"
-              disabled={settings.isOther}
+              disabled={isOtherOption}
             />
+            {isOtherOption && (
+              <Input 
```

Comment:
it's too much towards the right, would be good if we can keep it close to the option itself